Packs/PANOSPolicyOptimizer/Scripts/EntryWidgetPortBasedRules/EntryWidgetPortBasedRules.yml

```diff
@@ -0,0 +1,19 @@
+comment: Entry widget that returns the number of port based rules found by PAN-OS
+  policy optimizer.
+commonfields:
+  id: EntryWidgetPortBasedRules
+  version: -1
+dockerimage: demisto/python3:3.9.8.24399
+enabled: true
+name: EntryWidgetPortBasedRules
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- dynamic-section
+type: python
+fromversion: 5.5.0
```

Comment:
If the pack / layout only from version 6 - we should change the fromversion of all the widgets to be fromversion 6